### PR TITLE
4.0.dev25

### PR DESCRIPTION
**Pull request by michaelweinold**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by michaelweinold** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,24 +18,23 @@ build:
...
-    - requests >=1.1.0
-    - scipy
-    - stats_arrays
     - tqdm
     - voluptuous
     - whoosh
     - wrapt
+    - bw2parameters
+    - bw_processing
+    - bw-projects >=2.1.0
```
Comment (cmutel) [inline recipe/meta.yaml]: Sorry, we don't need this (bw_projects) anymore.

Comment (michaelweinold) [reply recipe/meta.yaml]: ...sorry, I forgot.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,24 +18,23 @@ build:
...
 requirements:
   host:
-    - python >=3.4
+    - python >=3.9
     - pip
     - setuptools
   run:
-    - bw2parameters
-    - bw_processing
-    - lxml
```
Comment (cmutel) [inline recipe/meta.yaml:27]: lxml removed?

Comment (michaelweinold) [reply recipe/meta.yaml:27]: No 🫤

**Pull request merged**